src/rally/api.clj

```diff
@@ -1,15 +1,26 @@
 (ns rally.api
-  (:require [clj-http.client :as client]
+  (:require [cheshire.core :as json]
+            [clj-http.client :as client]
             [clj-http.conn-mgr :as conn-mgr]
             [clj-http.cookies :as cookies]
+            [clojure.pprint :refer [pprint]]
+            [clojure.string :refer [lower-case]]
             [environ.core :as env]
             [rally.api.data :as data]
-            [rally.api.request :as request])
-  (:use [slingshot.slingshot :only [throw+]])
+            [rally.api.request :as request]
+            [slingshot.slingshot :refer [throw+]])
   (:refer-clojure :exclude [find]))
 
 (def ^:dynamic *current-user*)
 
+(def ^:dynamic *debug* false)
+
+(defn- debug? [request]
+  (or *debug*
+      (= (lower-case (str (:debug request))) "true")
```

Comment:
Why a string instead of a boolean?

Reply:
Because environ doesn't convert the environment variable into a boolean.  It seemed easier to deal with that here rather than in the config code.  And, kept the change set smaller.  I could be convinced otherwise.

Reply:
Thats fine if it only is populated via env var. I was thinking if it was part of a programatic api then a dev might be confused by it.